charts/cluster-api-runtime-extensions-nutanix/templates/ccm/nutanix/manifests/helm-addon-installation.yaml

```diff
@@ -16,6 +16,8 @@ data:
     prismCentralAdditionalTrustBundle: {{ `{{ printf "%q" . }}` }}
     {{ `{{- end }}` }}
 
+    ignoredNodeIPs: [ {{ printf "%q" .ControlPlaneEndpoint }} ]
```

Comment:
I am only guessing, but isnt the endpoint value including the port `x.x.x.x:9443` ?

Reply:
Its not, its the host part only...
```
ControlPlaneEndpoint:              nutanixConfig.ControlPlaneEndpoint.Host,
```

Maybe use a different name for the value?

Reply:
Changed the name.